src/transaction/flashback_cl.c

```diff
@@ -414,7 +414,33 @@ flashback_process_column_data (char **data, char **sql, int *max_sql_size, DB_TY
     case PACK_STRING:
       ptr = or_unpack_string_nocopy (ptr, &s_data);
 
-      if (IS_STRING_TYPE (type))
+      if (s_data == NULL)
+	{
+	  if (is_condition_column == true)
```

Comment:
How about just checking `sql - 2 == '='` without the `is_condition_column` parameter? it seems to me that the variable doesn't provide any flexibility because if the query format is changed, we should change how to deal with it too.

Reply:
or feel free to tell me the advantage of using the parameter that I missed. :)

Reply:
it seems simpler to check if it is a condition column.
it is easier to check `sql -2 == '='`, but as you mentioned, it could provide no flexibility and readability. 
Maybe, I think it is better to check if there is `where` in the sql statement.